src/Components/CookieConsentManager/categories.ts

```diff
@@ -0,0 +1,34 @@
+interface Category {
+  key: string
+  name: string
+  description: string
+}
+
+export const CATEGORIES: Category[] = [
+  {
+    key: "necessary",
+    name: "Strictly Necessary Cookies",
+    description:
+      "These cookies are required to enable core site functionality. They cannot be turned off.",
+  },
+  {
+    key: "functional",
+    name: "Functional Cookies",
+    description:
+      "These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.",
+  },
+  {
+    key: "performance",
+    name: "Performance Cookies",
+    description:
+      "These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site. If you do not allow these cookies we will not know when you have visited our site, and will not be able to monitor its performance.",
+  },
+]
+
```

Comment:
OneTrust has two [additional categories](https://www.notion.so/artsy/Cookie-Consent-01dace52a765476ea4e9353260ce9a7f?pvs=4#deb05d1a4f2c45f89e7c5ad54b0089f5): `Targeting`, `Social Media`. I think the way OneTrust works is that if there's no cookie for a category, their  Pref Center does not show that category for toggling. But it looks like the intent here is to display all categories?

Reply:
I suppose it's a decision to make. One benefit of displaying all categories is that we'd be able to imply consent for future destinations/integrations based on that category.

That said; where do we have advertising cookies on the site, since I see that category show up in the OneTrust dialog?

Reply:
looks like EU pref center has `Targeting` as a sub-category of `Advertising`. so it's really `Targeting` cookie that will cause `Advertising` category to be shown?
<img width="1119" alt="Screen Shot 2023-04-20 at 10 22 44 AM" src="https://user-images.githubusercontent.com/63004951/233396965-53054d30-b78d-4341-b45f-4ffad45cca33.png">

oof. so compilcated.

there are `Targeting` cookies in the [table](https://www.notion.so/artsy/e8b6666d69204f8c95c494b16c8d68d7?v=e7b597d613b14258844dde558432c50b&pvs=4). [`__gads`](https://www.notion.so/artsy/__gads-3e835fa674354af5844d77343fd648f4?pvs=4), for example, which is dropped on an [Article page](https://www.artsy.net/article/artsy-editorial-warhol-basquiat-ultimate-art-collab-joint-works-more).

Reply:
Well, those are internal ads — we're not re-targeting anyone with that. We can of course add the advertising category and use it to hide or show those ads completely but I'm not so sure it's needed?